functions/player.py

```diff
@@ -28,10 +29,13 @@ def update(self, parent, *args, **kwargs):
 		## Calculate the speed based of speed over time
 		## Same for turning.
 		speed_factor = time_last_render * (min(self.speed, self.max_speed)*self.multiplier)
-		turn_factor = time_last_render * self.turn_speed
+		if self.instant_turning:
+			turn_factor = self.turn_speed
+		else:
+			turn_factor = time_last_render * self.turn_speed
 
-		## 
-		#wself.target_angle = ((atan2(mouse_y-self.y, mouse_x-self.x)/pi*180)+360)%360
+		## Follow the mouse, use this code:
+		# wself.target_angle = ((atan2(mouse_y-self.y, mouse_x-self.x)/pi*180)+360)%360
```

Comment:
It's supposed to be `self.target_angle` not `wself.target_angle`. It's commented code so it won't affect anything. But perhaps fix this spelling error in the next commit.